### PR TITLE
Adds facebook and instagram to EmbedConsentBanner - WSTEAM1-283/284/286/287

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves WSTEAM1-283/284/286/287

**Overall change:**
- Adds Instagram and Facebook to Consent Banner
- Updates unit tests 

**Code changes:**

As above

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by amoore108 — APPROVED** [review]

**Review by eagerterrier — APPROVED** [review]
lgtm-ship-it

**Comment by amoore108** [comment]
@pvaliani @HarveyPeachey Once #10561 goes in, if you could remove the `if` statement here: 
https://github.com/bbc/simorgh/pull/10561/files#diff-2ee52a73896ea7933cb8079d72b9a2397d20452f4683c0b3763641b39062a06bR238-R240 
that should allow the E2Es to account for the consent banner for all social media providers.

The cypress commands should remain, just the if statement itself can go.

File: src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx [hunk 1]
```diff
@@ -384,6 +402,16 @@ describe('AmpSocialEmbed', () => {
...
+      const hasConsentBanner =
+        embed.oembed.provider_name === 'youtube' ||
+        embed.oembed.provider_name === 'facebook' ||
+        embed.oembed.provider_name === 'instagram';
+
+      if (hasConsentBanner) {
+        const button = screen.getByTestId('banner-button');
+        fireEvent.click(button);
+      }
+
```
Comment (amoore108) [inline src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx:414]: This check may be able to be removed? Can be a follow up change though if the tests pass.

**Review by amoore108 — APPROVED** [review]

File: src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx (hunk 1 above)
Comment (HarveyPeachey) [reply src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx:414]: Yeah we can do a follow up, this button invoking doesn't fix the test so needs to be looked at anyway

**Pull request merged**